File: Java/MochiKering.java
public class MochiKering extends ProdukMochi {
    private String rasa, dayaTahan, kemasan;

    public MochiKering(String kode, String nama, int harga, String rasa, String dayaTahan, String kemasan) {
        super(kode, nama, harga);
        this.rasa = rasa;
        this.dayaTahan = dayaTahan;
        this.kemasan = kemasan;
    }

    public String getRasa() { return rasa; }
    public String getDayaTahan() { return dayaTahan; }
    public String getKemasan() { return kemasan; }
}

File: Python/mochi_kering.py
from produk_mochi import ProdukMochi

class MochiKering(ProdukMochi):
    def __init__(self, kode, nama, harga, rasa, daya_tahan, kemasan):
        super().__init__(kode, nama, harga)
        self.rasa = rasa
        self.daya_tahan = daya_tahan
        self.kemasan = kemasan
